pylua/network.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+
+import asyncio
+import websockets
```

Comment:
@oherasym, i was talking aboyt websockets import. You import whole websocket first time, and just connect send time. First import is redudant. Imprort all required things using the second call only.